src/makeRobotAccountant.js

```diff
@@ -6,7 +6,13 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (firstNumber) =>
+    (secondNumber) =>
+      (++counter >= 4 && counter % 2 === 0)
```

Comment:
Got the idea, but let's make it more explicit:this increment is really easy to overlook here. 
Why don't we increase counter at the top of last function invocation and go with a simple `if` statement? 
Sure, it will not look that fancy, but readability always goes first.